src/modules/0.0.22/aws_memory_db/mappers/memory_db_cluster.ts

```diff
@@ -58,8 +64,8 @@ export class MemoryDBClusterMapper extends MapperBase<MemoryDBCluster> {
     out.securityGroups = securityGroups;
     out.status = cloudE.Status;
     out.subnetGroup =
-      (await awsMemoryDBModule.subnetGroup.db.read(ctx, `${cloudE.SubnetGroupName}|${region}`)) ??
-      (await awsMemoryDBModule.subnetGroup.cloud.read(ctx, `${cloudE.SubnetGroupName}|${region}`));
+      (await awsMemoryDBModule.subnetGroup.db.read(ctx, super.generateId(cloudE.SubnetGroupName, region))) ??
+      (await awsMemoryDBModule.subnetGroup.cloud.read(ctx, super.generateId(cloudE.SubnetGroupName, region)));
```

Comment:
Why not:

```suggestion
      (await this.module.subnetGroup.db.read(ctx, super.generateId(cloudE.SubnetGroupName, region))) ??
      (await this.module.subnetGroup.cloud.read(ctx, super.generateId(cloudE.SubnetGroupName, region)));
```

Reply:
(Also the `super.generateId` thing, but not going to make a suggestion on that in case I'm wrong about that)

Reply:
Oh, definitely, I did not take that much attention from the caller, I assumed it was fine. I'll update it